DOCSP-29504-many-to-one-clusters (#161)
mongodb/docs-cluster-to-cluster-sync: source/includes/migrationName-description.rst

Starting in ``mongosync`` 1.7.0, you can set a migration name for a sync

> operation. For example, set a migration name to identify the sync

Reviewer: ```suggestion
operation. For example, you can set a migration name to identify the sync
```

Author: Done.

Starting in ``mongosync`` 1.8.0, you can set a migration name for a sync

> operation. For example, you can set a migration name to identify the
> sync operations from multiple source clusters into one destination

Reviewer 2: ```suggestion
operation. For example, you can set a migration name to identify each
sync operation from multiple source clusters into one destination
```

Reviewer 2: The current phrasing is a bit unclear on whether a single migration name applies to all sync operations into one destination or each sync operation representing different source clusters.

Author: Done.
